pkg/api/v1/schema.go

```diff
@@ -628,12 +628,32 @@ type Interface struct {
 // +k8s:openapi-gen=true
 type InterfaceBindingMethod struct {
 	Bridge *InterfaceBridge `json:"bridge,omitempty"`
+	Slirp  *InterfaceSlirp  `json:"slirp,omitempty"`
```

Comment:
I think it should be name `Proxy` as it was agreed in the API doc.

Reply:
I Change it before but @fabiand tell me to change it.
I can change it again but we need to agree before I change it again.

Reply:
I think it's ok to leave the name slirp because is not a real proxy implementation.

Reply:
Yea, I'm not against it. I'm just not sure whether the term `slirp` helps people understand the way this interface will be connected. Perhaps, `user` or `userspace` is a better term?

Reply:
Let's please stick to `slirp` - it will help people to understand what it is about.
it's also the most precise name, and we probably come up with a more non-qemu solution where the term "userspace" or "proxy" is more suited.

As we discussed slirp is a step primarily to unblock, thus having this "weird" name here is fine - and as said we avoid naming conflicts with eventually more sophisticated solutions.

Btw - in the scope of the design doc slirp was not discussed, that's why it's fine to take this name.
I'll actually add slirp to the doc.